app/models/concerns/property_summarisable.rb

```diff
@@ -2,24 +2,24 @@ module PropertySummarisable
   # MAIN HOME
   # Value
   def main_home_value
-    from_cfe_payload("assessment.capital.capital_items.properties.main_home.value") if owns_property?
+    from_cfe_payload("assessment.capital.capital_items.properties.main_home.value") || 0
   end
 
   def smod_main_home_value
     main_home_value if house_in_dispute
```

Comment:
This, and all the other SMOD property fields, need a check whether there are any SMOD assets. If there are any SMOD assets, then for example this field should return a zero if the main home isn't SMOD. Otherwise it should return nil